src/display/editor/tools.js

```diff
@@ -1469,6 +1471,89 @@ class AnnotationEditorUIManager {
     }
   }
 
+  setUpDragSession() {
+    if (!this.hasSelection) {
+      return;
+    }
+    // Avoid to have spurious text selection in the text layer when dragging.
+    this.disableUserSelect(true);
+    this.#draggingEditors = new Map();
+    for (const editor of this.#selectedEditors) {
+      this.#draggingEditors.set(editor, [
+        editor.x,
+        editor.y,
+        editor.parent,
+        null,
+        null,
+        null,
+      ]);
```

Comment:
Can you please use *an Object* for the data that you're adding to the map, since looking at this it's somewhat difficult to understand exactly what all these properties are meant to represent?

(Even looking e.g. at the first loop in `endDragSession` below doesn't help all that much, in my opinion.)